torch/_inductor/pattern_matcher.py

```diff
@@ -280,7 +280,7 @@ def _match_fns(self, node: torch.fx.Node):
         return (
             isinstance(node, torch.fx.Node)
             and node.op == self.op
-            and node.target in self.fns_set
+            and extract_target(node) in self.fns_set
```

Comment:
why was this change needed ?

Reply:
This is used to support pattern matching for `call_module` node. For `call_module` node, `node.target` is a string and we need to extract the corresponding module for matching.